src/pal/src/exception/signal.cpp

```diff
@@ -600,7 +600,10 @@ static void common_signal_handler(int code, siginfo_t *siginfo, void *sigcontext
     // Fill context record with required information. from pal.h:
     // On non-Win32 platforms, the CONTEXT pointer in the
     // PEXCEPTION_POINTERS will contain at least the CONTEXT_CONTROL registers.
-    CONTEXTFromNativeContext(ucontext, &context, CONTEXT_CONTROL | CONTEXT_INTEGER | CONTEXT_FLOATING_POINT);
+    CONTEXTFromNativeContext(
+        ucontext,
+        &context,
+        CONTEXT_CONTROL | CONTEXT_INTEGER | CONTEXT_FLOATING_POINT | CONTEXT_XSTATE_YMMH);
```

Comment:
The CONTEXT_XSTATE_YMMH needs to be set only for _AMD64_. It would break ARM builds this way.